templates/hypershift-addon-manager-deployment.yaml

```diff
@@ -1,20 +1,21 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: {{ .Values.hypershiftAddonMgr.deploymentName }}
-  namespace: {{ .Values.installNamespace }}
+  name: {{ .Values.hypershiftAddonMgrOperator.name }}
+  namespace: {{ .Values.global.namespace }}
   labels:
-    app: {{ .Values.hypershiftAddonMgr.deploymentName }}
+    app: {{ .Values.hypershiftAddonMgrOperator.name }}
 spec:
-  replicas: {{ .Values.hypershiftAddonMgr.replicaCount }}
+  replicas: 1
```

Comment:
Can you confirm that there should always be a replicaCount of 1?
Does this deployment have leaderelection, and can/should it support HA mode if configured in the MCH?

Reply:
we might be able to make it HA, but this relaase, we might not have cycle for it...in this case, what's your suggestion?

Reply:
That is ok, we can leave as 1 for now and update once leaderelection is validated.
 
Currently we prefer if components support our HA toggle, but this is not a hard requirement for us now as far as I know.

We can always get this updated quickly if/when changes are made to support HA